src/components/Avatar/AvatarWrapper.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import classNames from 'classnames'
+
+
+const propTypes = {
+    tag: PropTypes.oneOfType([PropTypes.func, PropTypes.string]),
+    className: PropTypes.string,
+}
+
+const defaultProps = {
+    tag: 'div',
+}
+
+const AvatarWrapper = (props) => {
+    const {className, tag:Tag, extra, ...attributes} = props
+    const wrapperClass = classNames('avatar-wrapper',className,{
+        [`avatar-${extra}`]:extra
```

Comment:
perhaps it's better to include the `extra` string into the default class partial:

```js
[`avatar-extra-${extra}`]:extra
```

So the prop can be more explcit about the extra content:

```jsx
<AvatarWrapper extra="text">
...
</AvatarWrapper>
```